### PR TITLE
Add post upgrade Cb Check

### PR DESCRIPTION
**Pull request by monrog2**

### CODE REVIEW

**Comment by takishida** [comment]
Integration tests passed:

pytest outputs:
```
[Check  1/1] Post Upgrade Callback Integrity... Re-run script after APICs are upgraded and back to Fully-Fit  POST UPGRADE CHECK REQUIRED
[Check  1/1] Post Upgrade Callback Integrity... Re-run script after APICs are upgraded and back to Fully-Fit  POST UPGRADE CHECK REQUIRED
[Check  1/1] Post Upgrade Callback Integrity...                                                                                      PASS
[Check  1/1] Post Upgrade Callback Integrity...                                                                                      PASS
[Check  1/1] Post Upgrade Callback Integrity...                                                                   FAIL - OUTAGE WARNING!!
  Missed Objects       Impact
  --------------       ------
  infraImplicitSetPol  Infra implicit settings will not be deployed

  Recommended Action: Contact Cisco TAC with Output
  Reference Document: https://datacenter.github.io/ACI-Pre-Upgrade-Validation-Script/validations/#post-upgrade-callback-integrity


[Check  1/1] Post Upgrade Callback Integrity...                                                                                      PASS
[Check  1/1] Post Upgrade Callback Integrity...                                                                   FAIL - OUTAGE WARNING!!
  Missed Objects       Impact
  --------------       ------
  fvSlaDef             IPSLA monitor policy will not be deployed
  infraImplicitSetPol  Infra implicit settings will not be deployed

  Recommended Action: Contact Cisco TAC with Output
  Reference Document: https://datacenter.github.io/ACI-Pre-Upgrade-Validation-Script/validations/#post-upgrade-callback-integrity


[Check  1/1] Post Upgrade Callback Integrity...                                                                                      PASS
[Check  1/1] Post Upgrade Callback Integrity...                                                                   FAIL - OUTAGE WARNING!!
  Missed Objects                 Impact
  --------------                 ------
  fvSlaDef                       IPSLA monitor policy will not be deployed
  infraAssocEncapInstDef         VLAN for missing Mo will not be deployed to leaf
  infraImplicitSetPol            Infra implicit settings will not be deployed
  infraRsConnectivityProfileOpt  VPC for missing Mo will not be deployed to leaf

  Recommended Action: Contact Cisco TAC with Output
  Reference Document: https://datacenter.github.io/ACI-Pre-Upgrade-Validation-Script/validations/#post-upgrade-callback-integrity


[Check  1/1] Post Upgrade Callback Integrity...                                                                                      PASS
[Check  1/1] Post Upgrade Callback Integrity...                                                                   FAIL - OUTAGE WARNING!!
  Missed Objects                 Impact
  --------------                 ------
  compatSwitchHw                 64/32 bit image can unexpectedly pushed to switches
  fvSlaDef                       IPSLA monitor policy will not be deployed
  infraAssocEncapInstDef         VLAN for missing Mo will not be deployed to leaf
  infraImplicitSetPol            Infra implicit settings will not be deployed
  infraRsConnectivityProfileOpt  VPC for missing Mo will not be deployed to leaf

  Recommended Action: Contact Cisco TAC with Output
  Reference Document: https://datacenter.github.io/ACI-Pre-Upgrade-Validation-Script/validations/#post-upgrade-callback-integrity
```

File: aci-preupgrade-validation-script.py [hunk 1]
```diff
@@ -2960,6 +2961,81 @@ def access_untagged_check(index, total_checks, **kwargs):
...
+        },
+        "infraAssocEncapInstDef": {
+            "CreatedBy": "infraRsToEncapInstDef",
+            "SinceVersion": "5.2(4d)",
+            "Impact": "VLAN for missing Mo will not be deployed to leaf",
+        },
+        "compatSwitchHw": {
+            "CreatedBy": "",  # suppBit attribute is available from 6.0(2h)
+            "SinceVersion": "6.0(2h)",
+            "Impact": "64/32 bit image can unexpectedly pushed to switches",
```
Comment (monrog2) [inline aci-preupgrade-validation-script.py:3003]: "Impact": "64/32 bit image can unexpectedly pushed to switches",

to
`"Impact": "Unexpected 64/32 bit image can deploy to switches",`

**Review by monrog2 — COMMENTED** [review]
minor impact change, otherwise ok. will self-push then merge

**Review by monrog2 — COMMENTED** [review]
All good

**Pull request merged**